pulp/pulp.py

```diff
@@ -1218,7 +1218,7 @@ def __truediv__(self, other):
     def valid(self, eps: float = 0) -> bool:
         val = self.value()
         if self.sense == const.LpConstraintEQ:
-            return abs(val) <= eps
```

Comment:
Again, just hides the bug: https://github.com/coin-or/pulp/pull/807/files#diff-b2a258edfe0fa294f032a978656a4082842558faa98b0950d9f68c50f546bbe2R1370

Reply:
My links don't seem to have worked, there needs to be a check like:

```python
val = self.value()
if val is None:
    return False
```